lib/fog/ovirt/requests/compute/v4/vm_action.rb

```diff
@@ -0,0 +1,26 @@
+module Fog
+  module Compute
+    class Ovirt
+      class V4
+        class Real
+          def vm_action(options = {})
+            raise ArgumentError, "instance id is a required parameter" unless options.key? :id
+            raise ArgumentError, "action is a required parameter" unless options.key? :action
+
+            vm_service = client.system_service.vms_service.vm_service(options[:id])
+            vm_service.public_send(options[:action])
```

Comment:
@iNecas, this looks like a security issue to me - the parameter is not sanitized and sent directly to the object. I know this is not intended to be called directly by a user, but I think it deserves at least a warning comment in the code. I think the developer should be warned that the input is not sanitized inside, so it's his responsibility to sanitize the action before sending it to here.

Reply:
I don't think it's such a security issue because it's only public methods and we allow other actions like `destory_vm`.
We could change this to only allow power actions since that's why foreman core uses it but there might be other actions acceptable here/